help/implementation-playbook/best-practices/planning/valkey-service-configuration.md

```diff
@@ -147,7 +147,7 @@ stage:
 
 >[!NOTE]
 >
->In the previous example, the `full_page` cache is not relevant to Adobe Commerce on cloud infrastructure projects, because they use [Fastly](https://experienceleague.adobe.com/en/docs/commerce-on-cloud/user-guide/cdn/fastly).
+>In the previous example, the `full_page` cache is not relevant to Adobe Commerce on cloud infrastructure projects, because Commerce utilize [Fastly](https://experienceleague.adobe.com/en/docs/commerce-on-cloud/user-guide/cdn/fastly).
```

Comment:
```suggestion
>In the previous example, the `full_page` cache is not relevant to Adobe Commerce on cloud infrastructure projects, because those projects use [Fastly](https://experienceleague.adobe.com/en/docs/commerce-on-cloud/user-guide/cdn/fastly).
```